tensorboard/plugins/profile/profile_demo.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Dumps demo data for the profile dashboard.
+
+The data for profile plugin data is not generated by tensorflow workflow at this
```

Comment:
one of the goals for the demo is that it should show a new user how to setup the plugin. that isn't the case here. I guess this is because the APIs for writing the data aren't available yet - can you add a TODO to fix this once the tool is ready?